neo/sys/posix/platform_linux.cpp

```diff
@@ -56,21 +56,32 @@ Sys_EXEPath
 */
 const char* Sys_EXEPath()
 {
-	static char	buf[ 1024 ];
```

Comment:
PATH_MAX at least is POSIX (and honestly I doubt Hurd will ever be able to run doom3bfg), while the allocating functions like `get_current_dir_name()` are completely non-standard and thus a lot *less* portable.

But maybe we should use doom3's MAX_OSPATH and set that to something sane.
We have to anyway, because I think it's used all over the game to hold paths/filenames.
Setting it to PATH_MAX on unix-like systems (or maybe 4096 everywhere) is probably a sane default. I know some Linux filesystems support longer paths, but I don't really expect anyone to use paths longer than 4096 for their home-dir or their rbd3bfg install dir. (Right now it seems like 256 is even enough, as no one has complained yet, but it should really be higher)